src/web/client/extension.ts

```diff
@@ -200,9 +200,19 @@ export function activate(context: vscode.ExtensionContext): void {
 
     enableFileSearchFunctionality(portalsFS);
 
+    enableTextSearchFunctionality(portalsFS);
+
     showWalkthrough(context, WebExtensionContext.telemetry);
 }
 
+export function enableTextSearchFunctionality(portalsFS: PortalsFS) {
+    vscode.workspace.registerTextSearchProvider(PORTALS_URI_SCHEME, {
+        provideTextSearchResults: async (query, options, progress) => {
+            return portalsFS.searchTextResults(query, options, progress);
+        },
+    });
+}
+
```

Comment:
This registers a disposable, which you should dispose on deactivation.

Reply:
Not sure if I see the change for this on the latest commits. Could you provide a code pointer to the deactivation?

Reply:
@ritikramuka is this handled?